src/Common/tests/System/PlatformDetection.cs

```diff
@@ -18,9 +18,10 @@ public static partial class PlatformDetection
         // do it in a way that failures don't cascade.
         //
 
-        public static bool IsUap => IsWinRT || IsNetNative;
+        public static bool IsUap => IsInAppContainer || IsNetNative;
```

Comment:
See here for explanation: https://github.com/dotnet/corefx/issues/19907#issuecomment-302268922